packages/provider/src/util.ts

```diff
@@ -104,3 +105,18 @@ export function updateSolutions(solutions: pl.DataFrame, captchas: Captcha[], lo
         return captcha
     })
 }
+
```

Comment:
Not sure I understand this logic. If the task was running, wouldn't task.status=running, not completed?

Reply:
As discussed on slack, this logic was wrong. A unit test has been added.